aot/src/main/java/com/dylibso/chicory/experimental/aot/AotMethodInliner.java

```diff
@@ -31,16 +32,16 @@ public void visit(
                             String[] interfaces) {
                         super.visit(
                                 version,
-                                Opcodes.ACC_PRIVATE | Opcodes.ACC_FINAL,
-                                AotUtil.internalClassName(className + "$AotMethods"),
+                                Opcodes.ACC_FINAL | Opcodes.ACC_SUPER,
+                                internalClassName(className + "$AotMethods"),
                                 null,
                                 superName,
                                 null);
                     }
 
                     @Override
                     public void visitEnd() {
-                        visitNestHost(AotUtil.internalClassName(className));
```

Comment:
nit: do you have an IDE setting that does this change?
is it possible to enforce it, project wise, using spotless?
otherwise we are going to do forth and back when we edit the same files ...

Reply:
No, I do it manually.

Reply:
If you really care about it, please write a guideline somewhere, otherwise I think we are going to have a lot of changes in this detail... personally I don't care much ...